scriptworker/cot/verify.py

```diff
@@ -1281,6 +1281,12 @@ def verify_decision_command(decision_link, mach_commands=DECISION_MACH_COMMANDS)
         CoTError: on failure.
 
     """
+    if 'mozilla-mobile/focus-android' in get_source_url(parent_link):
+        log.warn(
+            'Repo "mozilla-mobile/focus-android" does not support JSON-e yet. Skipping parent task verifications'
```

Comment:
looks like we're not fully verifying by either cotv1 or cotv2?

Reply:
That's right. I think it's okay to leave it until https://github.com/taskcluster/taskcluster-github/pull/239 gets implemented. No APK can be published to users if the decision task gets altered.

Are you okay with it?

Reply:
We're not checking the decision task command, and we're not checking the decision task definition - how do we know if it's altered?

Reply:
> No APK can be published to users if the decision task gets altered.

My apologies, this wasn't very clear. I meant: if an attackers gets the right scopes and forges a decision task, then this person can't get the APK live (unless the attacker has access to Google Play too). There is no alteration verification per se, like you said.

Reply:
Ok. As long as the Focus team is aware of this hole and don't publish an apk without double checking that it's one they want to publish, this is probably ok.

Reply:
Actually, having a condition on the `cot_product` instead of the repo is safer. I'll use. Thanks for the hint!